hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -576,8 +576,17 @@ public void closeContainer(long containerID) throws IOException {
         .setTraceID(TracingUtil.exportCurrentSpan())
         .setContainerID(containerID)
         .build();
-    submitRequest(Type.CloseContainer,
-        builder -> builder.setScmCloseContainerRequest(request));
+    StorageContainerLocationProtocolProtos.SCMCloseContainerResponseProto
+        response = submitRequest(Type.CloseContainer,
+          builder -> builder.setScmCloseContainerRequest(
+            request)).getScmCloseContainerResponse();
+    if (response.getErrorCode().equals(
+        StorageContainerLocationProtocolProtos.SCMCloseContainerResponseProto.
+            Error.CONTAINER_ALREADY_CLOSED)) {
+      String errorMessage =
+          String.format("Container %s already closed", containerID);
+      throw new IOException(errorMessage);
+    }
```

Comment:
If changing `errorCode` to `optional`, check for `hasErrorCode()` here.

Nit: please import `SCMCloseContainerResponseProto` for brevity and to be consistent with other code in the same file.